pkg/omf/functions/cli/omf.cli.search.fish

```diff
@@ -1,26 +1,34 @@
-function omf.cli.search -d "CLI parser for the search command"
-
+function omf.cli.search -d 'Searches all available packages'
   switch (count $argv);
     case 1;
-      omf.search.pkg $argv[1]
+      echo (omf::under)"Packages"(omf::off)
```

Comment:
I see what you're saying, and I think I agree. I'll change things to say "plugin" when talking about a "non-theme package". Note that "omf search" called plugins "packages" already for a little while now (but omf.index.query doing so _is_ my fault).